src/summary/level-two-routes.component.tsx

```diff
@@ -8,44 +8,50 @@ import ProgramsLevelTwo from "./history/programs/programs-level-two.component";
 import MedicationsDetailedSummary from "./history/medications/medications-detailedSummary.component";
 import { AllergyCreateForm } from "./history/allergy-create-form.component";
 
-const levelTwoRoutes = [
+export const levelTwoRoutes = [
```

Comment:
We will need to handle the case of dynamically added widgets + routes. We can do that in subsequent PR.